### DISCUSSION TITLE
Archives for Product Attributes

### DISCUSSION THREAD

**Discussion by highheelcreative** [elementor/elementor#29216] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

We want to be able to do Product Archives for the attributes, we run a lot of car leasing websites for various clients and we nee dto create taxonomies to get around this when we didn't really need to.

### Describe the solution you'd like.

Product Archives to include Product Attributes

### Describe alternatives you've considered.

Create a Taxonomy to do the same job which wastes features of Woocommerce

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by beljp** [comment]
Hi everyone,

I just wanted to chime in to say I fully support this feature request. Having the ability to apply a custom archive template to product attribute pages would be incredibly useful for customizing the design and UX of WooCommerce stores.


Thanks for considering this improvement!
